app/models/invenio_rdm_record_converter.rb

```diff
@@ -594,4 +596,14 @@ def normalize_isbn(identifiers)
       end
     end
   end
+
+  def publisher
```

Comment:
Yeah.. let's go with the broader sweep right away:

```
if the record is a pediatric neurology record
  return "Pediatric Neurology Briefs Publishers"
else
  return first publisher
```

Did we use some logic in the past to identify pediatric neurology briefs papers e.g., for community mappings that we can re-use here? Otherwise: 
- if  the record was deposited by "Institutional Pnb" or 
- the record has the pediatric neurology brief DOI prefix 10.15844
seem like good indicators.

It's more reliable to return the hardcoded publisher value in pediatrics case. It's better to have a publisher value always.